rhobot.go

```diff
@@ -226,19 +232,42 @@ func healthcheckRunner(config *config.Config, healthcheckPath string, reportPath
 		_ = fhr.HandleReport(reader)
 	}
 
-	SMTPPortInt, _ := strconv.Atoi(config.SMTPPort)
-
 	// Email report
 	if emailListPath != "" {
-		ehr := report.EmailHandler{
-			SMTPHost:    config.SMTPHost,
-			SMTPPort:    SMTPPortInt,
-			SenderEmail: "-",
-			SenderName:  "-",
-			Subject:     "-",
-			Recipients:  []string{"-"},
-			HTML:        true,
+
+		df, err := report.ReadDistributionFormatYAMLFromFile(emailListPath)
+		if err != nil {
+			log.Fatal("Failed to read distribution format!")
 		}
-		_ = ehr.HandleReport(reader)
+
+		for _, level := range report.LogLevelArray {
```

Comment:
Question: could we not make Severity a map of string[[]string], and just loop over Severity?  like `for level, emails in df.Severity`?  Or would that break the YAML reader?  That seems cleaner than looping over each LogLevel, even if no distribution list exist for that log level, but if it makes reading the YAML more complicated it may not be worth it.